deps/rabbitmq_stream/src/rabbit_stream_reader.erl

```diff
@@ -949,6 +997,18 @@ open(cast, {force_event_refresh, Ref}, #statem_data{
     Connection2 = ensure_stats_timer(Connection1),
     {keep_state, StatemData#statem_data{connection = Connection2}}.
 
+close_sent(enter, _OldState, StateData) ->
+    {next_state, ?FUNCTION_NAME, StateData, {state_timeout, ?STATE_TIMEOUT, close}};
+close_sent(state_timeout, close, #statem_data{
+                                    transport = Transport,
+                                    connection = #stream_connection{socket = Socket} = Connection,
+                                    connection_state = State
+                                   }) ->
+    rabbit_log_connection:warning("Closing connection because of timeout in state '~s'.", [?FUNCTION_NAME]),
```

Comment:
It would be useful to indicate that this is likely due to a [lack of] client action.

Reply:
Added that indication, thank you.